core/src/main/java/org/openedx/core/Validator.kt

```diff
@@ -4,15 +4,19 @@ import java.util.regex.Pattern
 
 class Validator {
 
-    fun isEmailValid(email: String): Boolean {
-        val validEmailAddressRegex =
-            Pattern.compile("^[A-Z0-9._%+-]+@[A-Z0-9.-]+\\.[A-Z]{2,6}$", Pattern.CASE_INSENSITIVE)
-        val matcher = validEmailAddressRegex.matcher(email)
-        return matcher.find()
+    fun isEmailOrUserNameValid(email: String): Boolean {
```

Comment:
we are still only verifying the email address, do we need to verify the username?

Reply:
it would be great if you could just check `space` in the field.

Reply:
already replaced with `isBlank`, so no need to check white spaces. Also can trim the white space in case of non-blank. thoughts?